backend/package.json

```diff
@@ -33,15 +33,14 @@
     "cors": "^2.8.5",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "faker": "^5.1.0",
     "joi": "^17.3.0",
     "jsonwebtoken": "^8.5.1",
-    "mocha": "^8.2.1",
     "mongoose": "^5.10.14",
     "mongoose-paginate-v2": "^1.3.11",
     "morgan": "^1.10.0",
     "should": "^13.2.3",
     "sinon": "^9.2.1",
-    "supertest": "^6.0.1",
```

Comment:
why you've removed supertest and mocha libs?

Reply:
![image](https://user-images.githubusercontent.com/62155886/99543805-43062a00-29bc-11eb-99b6-eedb5c3ca33e.png)

Reply:
This libs are here.